res/values/strings.xml

```diff
@@ -413,6 +413,9 @@
     <string name="start_app">Start…</string>
     <!-- this is a warning that is shown when one tries to send something to a chat that is not yet accepted. -->
     <string name="accept_request_first">Please accept the chat request first.</string>
+    <!-- placeholder will be replaced by a file name -->
+    <string name="send_file_to">Send \"%1%s\" to…</string>
```

Comment:
will there be there some cutting if the file name is very long?

Reply:
also files shared from other apps sometimes don't even have a filename, this might introduce some weird names on display from data shared from other apps

Reply:
> will there be there some cutting if the file name is very long?

that could be done by the implementation. but even if it is not done and the text is cut at the end, sth. as `Send "this-is-a-very-long-filename-that...` still seems better than `Share with`

> also files shared from other apps sometimes don't even have a filename

this is only about sending messages from webxdc, and there we require a file name.